network/src/main/java/uk/gov/android/network/api/ApiRequest.kt

```diff
@@ -2,21 +2,24 @@ package uk.gov.android.network.api
 
 import uk.gov.android.network.client.ContentType
 
-sealed class ApiRequest {
+sealed class ApiRequest(
+    var apiHeaders: List<Pair<String, String>> = emptyList()
```

Comment:
why or be consistent with the previous vals and keep this variable named `headers` as well?
```suggestion
    val headers: List<Pair<String, String>> = emptyList()
```
We already have Api wording in the `ApiRequest` so its obvious this headers are part of the API contract.

Also, any reasons why keeping it `var` rather than `val`? I think its better if we keep this type immutable.

Reply:
As discussed ApiRequest now reverted to original structure but with headers field in FormUrlEncoded data class